ui/DesignSystem/Component/SidebarLayout.svelte

```diff
@@ -1,12 +1,16 @@
-<script lang="ts">
+<script lang="typescript">
   import { getContext } from "svelte";
 
+  import { fallback } from "../../src/identity";
+  import type { Authenticated } from "../../src/session";
+
   import Sidebar from "./Sidebar.svelte";
 
   export let dataCy = "";
   export let style = "";
 
-  const session = getContext("session");
+  const session: Authenticated = getContext("session");
+  const identity = session.identity || fallback;
```

Comment:
If `session` is of type `Authenticated`, then `session.identity` is always set. But in actuality, `session` can be of type `Unauthenticated`. In this case We should probably not use the fallback but rather throw or show an error.

Reply:
Was a leftover, thanks for spotting it. I introduced the vairants after this fallback was in place.